platform/android/src/ch/opengis/qfield/QFieldActivity.java

```diff
@@ -396,36 +343,47 @@ public void run() {
         });
     }
 
-    private double statusBarMargin() {
-        double margin = sceneTopMargin;
-        if (margin > 0) {
-            int resourceId = getResources().getIdentifier("status_bar_height",
-                                                          "dimen", "android");
-            if (resourceId > 0) {
-                double dimension = getResources().getDimension(resourceId);
-                if (dimension > 0) {
-                    margin = dimension;
-                }
+    private Insets getSafeInsets() {
```

Comment:
A note on this function: we'll be able to get rid of this one altogether when we move to Qt >= 6.9, where we finally have a safeAreaMargins implementation for Android (see https://bugreports.qt.io/browse/QTBUG-131519).  We already use QPlatformWindow's safeAreaMargins on iOS and it'll be nice to use it for Android too. 

That said, this is for a much latter date. Just stating for the record :)